src/VehicleInterface.c

```diff
@@ -399,6 +400,151 @@ void vsc_send_heartbeat(VscInterfaceType* vscInterface, uint8_t EStopStatus) {
 
 }
 
+/** vsc_scm_target_set()
+ * @brief This function sends a SCM Target Set Command to the VSC to prepare for a settings grab.
+ * 
+ * @param vscInterface VSC Interface Structure.
+ * @param target_id The identifier for the target, 0 for local.
+ */
+void vsc_scm_target_set(VscInterfaceType* vscInterface, int target_id)
+{
+	VscMsgType scmTargetSetMsg;
+	SCMTargetSetMsgType *msgPtr = (SCMTargetSetMsgType*) scmTargetSetMsg.msg.data;
+
+	/* Fill Message */
+	scmTargetSetMsg.msg.msgType = MSG_SCM_TARGET_SET;
+	scmTargetSetMsg.msg.length = sizeof(SCMTargetSetMsgType);
+
+	/* Fill in the Target ID */
+	msgPtr->target_id = target_id;
+
+	/* Send Message */
+	if (vsc_send_msg(vscInterface, &scmTargetSetMsg) < 0) {
+    // Error print commented out due to spam
+	  // fprintf(stderr, "vsc_example: Send Message Failure (Errno: %i)\n", errno);
+	}
+}
+
+/** vsc_scm_target_get()
+ * @brief This function sends a SCM Target Get Command to the VSC to prepare for a settings grab.
+ * 
+ * @param vscInterface VSC Interface Structure.
+ */
+void vsc_scm_target_get(VscInterfaceType* vscInterface)
+{
+	VscMsgType scmTargetGetMsg;
+	SCMTargetGetMsgType *msgPtr = (SCMTargetGetMsgType*) scmTargetGetMsg.msg.data;
+
+	/* Fill Message */
+	scmTargetGetMsg.msg.msgType = MSG_SCM_TARGET_GET;
+	scmTargetGetMsg.msg.length = sizeof(SCMTargetGetMsgType);
+
+	/* Send Message */
+	if (vsc_send_msg(vscInterface, &scmTargetGetMsg) < 0) {
+    // Error print commented out due to spam
+	  // fprintf(stderr, "vsc_example: Send Message Failure (Errno: %i)\n", errno);
+	}
```

Comment:
Same as above, will the commented out message be addressed in the future? Additionally, I see this happens several other times. So the same goes to these other instances.

Reply:
Same as above, will defer to the previous developer's judgement on this.